Examples/Monocular/mono_kitti.cc

```diff
@@ -32,25 +32,34 @@
 using namespace std;
 
 void LoadImages(const string &strSequence, vector<string> &vstrImageFilenames,
-                vector<double> &vTimestamps);
+                vector<double> &vTimestamps, int start_frame=0, int end_frame=-1);
 
 int main(int argc, char **argv)
 {
-    if(argc != 5)
+    if(argc <= 5)
     {
-        cerr << endl << "Usage: ./mono_kitti path_to_vocabulary path_to_settings path_to_sequence path_to_output_file" << endl;
+        cerr << endl << "Usage: ./mono_kitti path_to_vocabulary path_to_settings path_to_sequence path_to_output_file <start frame> <end frame>" << endl;
         return 1;
     }
 
+    int start_frame = 0;
+    int end_frame = -1;
+    if(argc >= 6) {
+        start_frame = std::stoi(argv[5]);
+    } 
+    if(argc >= 7) {
+        end_frame = std::stoi(argv[6]);
+    }
+
     // Retrieve paths to images
     vector<string> vstrImageFilenames;
     vector<double> vTimestamps;
-    LoadImages(string(argv[3]), vstrImageFilenames, vTimestamps);
+    LoadImages(string(argv[3]), vstrImageFilenames, vTimestamps, start_frame, end_frame);
 
     int nImages = vstrImageFilenames.size();
 
     // Create SLAM system. It initializes all system threads and gets ready to process frames.
-    ORB_SLAM2::System SLAM(argv[1],argv[2],ORB_SLAM2::System::MONOCULAR,false);
+    ORB_SLAM2::System SLAM(argv[1],argv[2],ORB_SLAM2::System::MONOCULAR,true);
```

Comment:
We should turn the viewer back off for running headless

Reply:
Fixed. Changed back to `false`.